docs/en/connector-v2/source/AmazonDynamoDB.md

```diff
@@ -17,41 +17,41 @@ Read data from Amazondynamodb.
 
 ## Options
 
-| name             | type   | required | default value |
-| ---------------- | ------ | -------- | ------------- |
-| url              | string | yes      | -             |
-| region           | string | yes      | -             |
-| access_key_id    | string | yes      | -             |
-| secret_access_key| string | yes      | -             |
-| table    	       | string | yes      | -             |
-| schema           | object | yes      | -             |
-| common-options   |        | yes      | -             |
+| name              | type   | required | default value |
+|-------------------|--------|----------|---------------|
+| url               | string | yes      | -             |
+| region            | string | yes      | -             |
+| access_key_id     | string | yes      | -             |
+| secret_access_key | string | yes      | -             |
+| table             | string | yes      | -             |
+| schema            | object | yes      | -             |
```

Comment:
`schema` type is best modified to `config`

Reply:
Done